fontir/src/orchestration.rs

```diff
@@ -119,22 +340,29 @@ pub type IrWork = dyn Work<Context, WorkId, WorkError> + Send;
 pub struct Context {
     pub flags: Flags,
 
-    pub(crate) paths: Arc<Paths>,
+    pub(crate) irio: Arc<FeIrIo>,
 
     // The input we're working on. Note that change detection may mean we only process
     // a subset of the full input.
     pub input: Arc<Input>,
 
-    acl: AccessControlList<WorkId>,
+    acl: Arc<AccessControlList<WorkId>>,
 
     // work results we've completed or restored from disk
     // We create individual caches so we can return typed results from get fns
-    init_static_metadata: ContextItem<ir::StaticMetadata>,
-    final_static_metadata: ContextItem<ir::StaticMetadata>,
-    global_metrics: ContextItem<ir::GlobalMetrics>,
-    glyph_ir: Arc<RwLock<HashMap<GlyphName, Arc<ir::Glyph>>>>,
-    feature_ir: ContextItem<ir::Features>,
-    kerning: ContextItem<ir::Kerning>,
+    pub static_metadata: ContextItem<WorkId, ir::StaticMetadata, FeIrIo>,
```

Comment:
okay, nice to see how this fits together.

Looking at how similar these all are, would it make sense to have something like,

```rust
type FeContextItem<T> = ContextItem<WorkId, T, FeIoFum>;
```
?

Reply:
I like it!